DependencyInjection/Configuration.php

```diff
@@ -44,7 +44,7 @@ public function __construct(array $resolversFactories, array $loadersFactories)
     }
 
     /**
-     * {@inheritdoc}
+     * @return TreeBuilder
```

Comment:
can we avoid this change? i dislike adding redundancy in phpdoc. 

or are the changes in this PR narrowing down the return types?

Reply:
The thing is that in Symfony 5.4, you get these deprecations: https://github.com/franmomu/LiipImagineBundle/runs/4102778452?check_suite_focus=true#step:11:124

here it is explained https://wouterj.nl/2021/09/symfony-6-native-typing#symfony-bundle-maintainers-compatibility-plan

Reply:
ah, i see. do our supported PHP versions allow to declare the return type even if the base class does not have it? then i would do that rather. at least for the Configuration class this should be reasonably safe.

if that is not possible, i'd rather supress those warnings in 2.x and change it in 3.x only, where we drop support for older PHP versions as well.

Reply:
Apparently it does: https://3v4l.org/vMpRt#v7.1.0

I've also changed the one in `LiipImagineExtension` that should be also safe too I think.

Reply:
thanks